src/ResourceManagement/RecoveryServices.Backup/RecoveryServices.Tests/app.config

```diff
@@ -42,7 +42,20 @@
     <add key="StorageAccountId" value="/subscriptions/f2edfd5d-5496-4683-b94f-b3588c579009/resourceGroups/shraccanrg/providers/Microsoft.Storage/storageAccounts/shraccanseastorv2" />
```

Comment:
These should beremoved.  Others running these tests will not have access to this account.  Again., it is better to do account setup in the test setup.